.github/workflows/python-package.yml

```diff
@@ -106,10 +103,13 @@ jobs:
       with:
         cuda: ${{ matrix.cuda_version }}
         method: 'local'
-        # sub-packages: '["nvcc","cudart","nvrtc_dev","cublas_dev","cusparse_dev","visual_studio_integration"]'
-    - name: Add msbuild to PATH
-      uses: microsoft/setup-msbuild@v1.1
-      if: ${{ startsWith(matrix.os, 'windows') }}
+        sub-packages: '["nvcc","cudart","cusparse","cublas","thrust","nvrtc_dev","cublas_dev","cusparse_dev"]'
```

Comment:
Maybe it is quicker to also do a network install here to avoid downloading the entire installer now that it's not cached?

Reply:
I guess, the local cache of the `cuda-toolkit*exe` appears to be causing some issues due to its large size. maybe `network` will work with `use-github-cache: true`,

Reply:
it can get more speed with `'network'`. thanks!

`use-github-cache: true` also not work with `'network'`. it seems like a bug of `Jimver/cuda-toolkit`.